_layouts/next-content.html

```diff
@@ -10,7 +10,7 @@
   {% assign page_type = 'page' %}
 {% endif %}
 
-<header class="next-hero next-hero--{{ page_type }} next-hero--{{ page.slug }} bg-{{ page.hero_color }} usa-dark-background">
+<section class="next-hero next-hero--{{ page_type }} next-hero--{{ page.slug }} bg-{{ page.hero_color }} usa-dark-background" aria-label="Page description">
```

Comment:
Use `section` for this landmark to avoid duplication with the site `header` above